src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -22,6 +29,13 @@ export class ExamChecklistComponent implements OnChanges, OnInit, OnDestroy {
     hasOptionalExercises = false;
     countMandatoryExercises = 0;
     isTestExam: boolean;
+    isEvaluatingQuizExercises: boolean;
+    isAssessingUnsubmittedExams: boolean;
+    existsUnfinishedAssessments: boolean = false;
+    existsUnassessedQuizzes: boolean = false;
+    existsUnsubmittedExercises: boolean = false;
+    isExamOver = false;
```

Comment:
Can you keep it consistent with the type annotations?
I think most of the time we either use a type annotation or a default value

Reply:
Fixed